build.rs

```diff
@@ -4003,16 +4003,15 @@ mod c {
 
     /// Compile intrinsics from the compiler-rt C source code
     pub fn compile(llvm_target: &[&str]) {
-        let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
-        let target_env = env::var("CARGO_CFG_TARGET_ENV").unwrap();
-        let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap();
```

Comment:
These all need to stay as environment variables due to cross compilation

Reply:
Got it, thanks